librz/core/tui/panels.c

```diff
@@ -3010,25 +3013,32 @@ void __init_panel_param(RzCore *core, RzPanel *p, const char *title, const char
 	v->refresh = true;
 	v->edge = 0;
 	if (title) {
+		char *tmp = rz_str_dup(title);
 		free(m->title);
-		m->title = rz_str_dup(title);
+		m->title = tmp;
 		if (cmd) {
+			char *tmp = rz_str_dup(cmd);
 			free(m->cmd);
-			m->cmd = rz_str_dup(cmd);
+			m->cmd = tmp;
 		} else {
+			char *tmp = rz_str_dup("");
 			free(m->cmd);
-			m->cmd = rz_str_dup("");
+			m->cmd = tmp;
 		}
 	} else if (cmd) {
+		char *tmp1 = rz_str_dup(cmd);
 		free(m->title);
-		m->title = rz_str_dup(cmd);
+		m->title = tmp1;
+		char *tmp2 = rz_str_dup(cmd);
 		free(m->cmd);
-		m->cmd = rz_str_dup(cmd);
+		m->cmd = tmp2;
 	} else {
+		char *tmp1 = rz_str_dup("");
 		free(m->title);
-		m->title = rz_str_dup("");
+		m->title = tmp1;
+		char *tmp2 = rz_str_dup("");
 		free(m->cmd);
-		m->cmd = rz_str_dup("");
+		m->cmd = tmp2;
```

Comment:
Same here. define it once within the if and then reuse it.

Reply:
Okayy